src/README.md

```diff
@@ -63,22 +63,22 @@ The libdaos and libdfs libraries provide the foundation to support domain-specif
 <a id="13"></a>
 ### Agent
 
-The <a href="control/agent/README.md">DAOS agent</a> is a daemon residing on the client node and interacts with the DAOS client library through dRPC to authenticate the application process. It is a trusted entity can sign the DAOS Client credentials using local certificates. The agent can support different authentication frameworks and uses an Unix Domain Socket by default. The DAOS agent is written in Go and communicates through gRPC with the control plane component of each DAOS server to provide DAOS system membership information to the client library and to support pool listing.
+The <a href="control/agent/README.md">DAOS agent</a> is a daemon residing on the client node and interacts with the DAOS client library through dRPC to authenticate the application process. It is a trusted entity that can sign the DAOS Client credentials using local certificates. The agent can support different authentication frameworks and uses a Unix Domain Socket to communicate with the client library. The DAOS agent is written in Go and communicates through gRPC with the control plane component of each DAOS server to provide DAOS system membership information to the client library and to support pool listing.
 
 <a id="2"></a>
-## Network Transport & Communications
+## Network Transport and Communications
 
 As introduced in the previous section, DAOS uses three different communication channels.
 
 <a id="21"></a>
-### gRPC & Protocol Buffers
+### gRPC and Protocol Buffers
 
 gRPC provides a bi-directional secured channel for DAOS management. It relies on TLS/SSL to authenticate the agent daemon, the administrator role and the servers. Protocol buffers are used for RPC serialization and all proto files are located in the [proto](proto) directory.
 
 <a id="22"></a>
 ### dRPC
 
-dRPC is communication channel built over Unix Domain Socket that is used for inter-process communications. It provides both a [C](common/dprc.c) and [Go](control/drpc) interface to support interactions between:
+dRPC is communication channel built over Unix Domain Socket that is used for inter-process communications. It provides both a [C](common/drpc.c) and [Go](control/drpc) interface to support interactions between:
```

Comment:
There is a README.md in the src/common directory that includes some additional info about the C API. Not sure if you wanted to link to that as well.